coalib/bearlib/languages/documentation/DocstyleDefinition.py

```diff
@@ -14,9 +14,11 @@ class DocstyleDefinition:
     documentation comment (for which language, documentation style/tool used
     etc.).
     """
+    Metadata = namedtuple("Metadata", "param_start, param_end, return_sep")
```

Comment:
though `namedtuple` allows it, I'm not a fan of using a single string to specify attribute names. On module load they get splitted anyway (though the performance impact is minor, but there is one), so let's do this ourselves with using a tuple `("param_start", "param_end", "return_sep")`. Imo the single string function exists for easier typing in the interactive shell^^

But: This is just my preference. If you want you can keep this :P